autobot-backend/llc/api/agent_api.py

```
@@ -73,9 +75,42 @@ class StatusUpdate(BaseModel):
+        new_status = WorkItemStatus(body.status)
+    except ValueError as exc:
+        raise HTTPException(status_code=422, detail=f"Unknown work item status {body.status!r}") from exc
+
+    factory = get_async_session_factory()
+    async with factory() as session:
+        try:
+            item = await lazy_singleton(WorkItemService)().transition_status(
+                session,
+                work_item_id=str(item_id),
+                new_status=new_status,
+                company_id=str(company_id),
```

Ruff (0.16.3)

[error] Undefined name `company_id`

(F821)